index.html

```diff
@@ -2,11 +2,127 @@
 <html lang="en-us">
   <head>
     <meta charset="UTF-8" />
-
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+      integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
+      crossorigin="anonymous"
+    />
     <link rel="stylesheet" href="style.css" />
- 
-    <title>Sing Up</title>
+    <title>Intuit Accounts - Sign Up</title>
   </head>
-<body>
-</body>
+
+  <body>
+    <nav class="nav-bar">
+      <div class="header-container">
+        <div class="left-header">
+          <p>Already have an Intuit account?</p>
+          <a href="#"><p>I forgot my User ID or Password</p></a>
+        </div>
+        <span>
+          <button type="button"><i class="fa fa-lock" aria-hidden="true"></i> Sign In</button>
+        </span>
+      </div>
+    </nav>
+
+    <header>
+      <div>
+        <a href="#"><img src="images/image.png" alt="image" /></a>
+      </div>
+      <div class="tools">
+        <a href="#"
+          ><img
+            class="icons"
+            src="https://plugin.intuitcdn.net/identity-authn-core-ui/images/mint-favicon-4accf8f4.svg"
+            alt="image"
+        /></a>
+        <a href="#"><span class="icon-names">mint</span></a>
+        <a href="#"
+          ><img
+            class="icons"
+            src="https://plugin.intuitcdn.net/identity-authn-core-ui/images/quickbooks-favicon@2x-c89488e8.svg"
+            alt="image"
+        /></a>
+        <a href="#"><span class="icon-names">quickbooks</span></a>
+        <a href="#"
+          ><img
+            class="icons"
+            src="https://plugin.intuitcdn.net/identity-authn-core-ui/images/tt-favicon@2x-fbd64ab8.svg"
+            alt="image"
+        /></a>
+        <a href="#"><span class="icon-names">turbotax</span></a>
+      </div>
+    </header>
+
+    <main>
+      <div>
+        <div class="box">
+          <div class="title-box">
+            <span class="title">Create an Intuit account <br /><br /></span>
+            <span class="sub-text"
+              >One account for everything Intuit, including Mint. <br /><a
+                href="#"
+                >Learn More</a
+              >
+            </span>
+          </div>
+
+          <div class="info-block">
+            <label class="mail" >Email address</label>
+            <input
+              type="email"
+              name="email"
+              
+              class="email"
+              required
+            />
+          </div>
+          <div class="info-block">
+            <label class="mail" >Phone (recommended)</label>
+            <input type="text" name="email" class="email" required />
+          </div>
```

Comment:
"phone" should have an `<input>`of type `tel`

Reply:
Issue corrected, thank you!